aeron-client/src/main/c/util/aeron_clock.c

```diff
@@ -29,46 +29,21 @@
 int64_t aeron_nano_clock()
 {
     struct timespec ts;
-#if defined(__CYGWIN__) || defined(__linux__)
```

Comment:
These should not call into aeron_windows.c for there clock. And should be put back so that they directly call the system calls.

Reply:
Agreed, updated.